### ISSUE TITLE
Decision details chart with overlapping text

### ISSUE THREAD

**Issue by bingen** [levelkdev/futarchy-app#113]
In the header (Yes / No) and in the dates on X axis:

![Screenshot from 2019-05-09 10-48-36](https://user-images.githubusercontent.com/701095/57440378-3261e900-7248-11e9-8230-10bea8dc54e3.png)

**Comment by emaG3m** [comment]
I'm trying to reproduce this. My Charts are looking a-ok. Do you have any ideas what throws the chart off? 
<img width="393" alt="Screen Shot 2019-05-10 at 2 15 48 PM" src="https://user-images.githubusercontent.com/5539720/57548155-4122cb80-732e-11e9-8243-16dab3fb537b.png">

**Comment by bingen** [comment]
I don't know, but I was using Brave Version 0.62.51 Chromium: 73.0.3683.103 (Official Build) (64-bit), if that helps.
In Firefox works fine.

**Comment by emaG3m** [comment]
thanks!!

**Comment by jpkcambridge** [comment]
This appears to be a problem caused by Brave blocking 3rd party device recognition and that impacting the rendering of the chart with Chart.js

<img width="643" alt="Screen Shot 2019-05-15 at 4 53 11 PM" src="https://user-images.githubusercontent.com/874683/57808760-0ce64b00-7732-11e9-848d-719691255fd6.png">

**Comment by bingen** [comment]
Oh, good catch! Feel free to close it then. Thanks!

**Comment by jpkcambridge** [comment]
Well, it will probably still be an issue to people using Brave.  There might not be an easy fix regarding how we use Chart.js but maybe makes sense to alert users somehow?

**Comment by bingen** [comment]
Yes, absolutely, that's why I didn't close it myself ;-) But I also see it's not an easy fix nor a highly critical issue, so your call.
